src/components/app-top-bar.js

```diff
@@ -220,6 +220,11 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
     function hideParameters() {
         setShowParameters(false);
     }
+    function formatTreeNodeLabel(selectedTreeNode) {
```

Comment:
Maybe give the string selectedTreeNode.data.label to this method and not the object